src/pdc.h

```diff
@@ -1,10 +1,10 @@
+#include <string>
+
 #define ADD 	'+'
 #define SUBSTRACT	'-'
 #define MULTIPLY	'*'
 #define DIVIDE	'/'
 
-//TODO: Dynamic typing, for now every value is int
-#define PseudoValue int // To enable dynamic typing we need to use one type for every value
+#define PseudoValue std::string // To enable dynamic typing we need to use one type for every value
```

Comment:
What about `* char` instead of string

Reply:
With string it's easier to convert to int :laughing:

Reply:
And in reverse